localstack/services/install.py

```diff
@@ -366,6 +396,59 @@ def install_lambda_java_libs():
         download(URL_LOCALSTACK_FAT_JAR, INSTALL_PATH_LOCALSTACK_FAT_JAR)
 
 
+def install_go_lambda_runtime():
+    install_glibc_for_alpine()
+
+    if not os.path.isfile(GO_LAMBDA_RUNTIME):
+        log_install_msg("Installing golang runtime")
+        response = requests.get(GO_RUNTIME_DOWNLOAD_URL, allow_redirects=True)
+        if not response.ok:
+            raise ValueError("golang runtime zip not downloaded")
+
+        file_location = config.TMP_FOLDER + "/" + GO_ZIP_NAME
+        open(file_location, "wb").write(response.content)
+
+        if not zipfile.is_zipfile(file_location):
+            raise ValueError("Downloaded file is not zip ")
+
+        zipfile.ZipFile(file_location).extractall(config.TMP_FOLDER)
+        st = os.stat(GO_LAMBDA_RUNTIME)
+        os.chmod(GO_LAMBDA_RUNTIME, st.st_mode | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH)
+
+        st = os.stat(GO_LAMBDA_MOCKSERVER)
+        os.chmod(GO_LAMBDA_MOCKSERVER, st.st_mode | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH)
+
+
+def install_glibc_for_alpine():
+    try:
+        run("apk info glibc")
+        return
+    except Exception:
+        pass
+
+    log_install_msg("Installing glibc")
+    try:
+        try:
+            run("apk add %s" % CA_CERTIFICATES)
+        except Exception:
+            raise Exception("ca-certificates not installed")
+
+        response = requests.get(GLIBC_KEY_URL, allow_redirects=True)
```

Comment:
nitpick: We could slightly simplify by using the `download(..)` util function from `utils/common.py` here:
```
download(GLIBC_KEY_URL, GLIBC_KEY)
download(GLIBC_URL, GLIBC_PATH)
```